tools/protovalidate-conformance/config.go

```diff
@@ -41,6 +43,8 @@ type config struct {
 	benchmark          int
 	cmd                string
 	args               []string
+	skippedFile        string
+	skippedTests       map[string][]string
 }
 
 func parseFlags() config {
```

Comment:
The function now has a case where it might not be able to properly create a valid `config`, it should be updated to return `(config, error)` because of this.

Our Go coding standards call on not using log.Fatal or its equivalents, and this function should return the error instead and the callers of this function should be updated.